app/views/govuk_component/related_items.raw.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  sections ||= []
+%>
+<aside class="govuk-related-items" role="complementary">
+  <% sections.each do |section| %>
+    <% section_title = section[:title] || "Elsewhere on GOV.UK" %>
```

Comment:
Lets remove it. If we need to add it back when we use this on real GOV.UK pages we can.